src/BMW_E65.cpp

```diff
@@ -253,9 +255,41 @@ void BMW_E65::SendAbsDscMessages(bool Brake_In)
 {
 
 //////////send abs/dsc messages////////////////////////
-    uint8_t a8_brake;
     uint8_t bytes[8];
 
+    uint16_t RPM_A = 0;
+    if (Ready())
+    {
+        RPM_A = 750 * 4;
```

Comment:
This breaks the rev counter needs replacing with        
`RPM_A = MAX(750, revCounter) * 4;`